scripts/apps/authoring/track-changes/views/suggestions-widget.html

```diff
@@ -10,12 +10,20 @@
             </figure>
             <div>{{users[item.suggestionInfo.author].display_name || users[item.suggestionInfo.author].username}}</div>
             <div class="date" sd-reldate ng-model="item.suggestionInfo.date"></div>
-            <div class="content">
+            <div class="content" ng-show="!item.oldText">
```

Comment:
It would be more clear to use if/else by `item.suggestionInfo.type === 'replace'`